sdcm/reporting/tooling_reporter.py

```diff
@@ -108,3 +137,18 @@ def __init__(self, driver_version: str, runner: CommandRunner, command_prefix: s
 
     def _collect_version_info(self) -> None:
         pass
+
+
+class CqlStressRustDriverVersionReporter(ToolReporterBase):
+    # pylint: disable=too-few-public-methods
+    TOOL_NAME = "rust-driver"
```

Comment:
It must support some suffix or prefix tweaking for distinguishing rust driver versions from `cql-stress` and `latte`.

Reply:
I didn't think about Latte here - see the name of the class `CqlStressRustDriverVersionReporter`
When introducing it for Latte, one may use this class as a base

Reply:
for c-s it was `TOOL_NAME = "cassandra-stress"`

so this should follow the same pattern

yes each one of them might report about multiple packages, but it's per tool

Reply:
I renamed `CqlStressRustDriverVersionReporter` -> `RustDriverVersionReporter`